de-DE/docs/find.ndb

```diff
@@ -183,20 +183,21 @@ getlasterror()~Gibt den letzten von einem try-catch-Block verarbeiteten Fehler z
 getmatchingparens(STR)~Gibt die Position der passenen schließenden Klammer in STR zurück~klammer zu geschlossene geschweifte getmatchingparens() closing parenthesis bracket curly brace~getmatchingparens(
 getompthreads()~Gibt die Zahl der maximal möglichen OpenMP-Threads zurück~CPU core logical logische kerne prozessor multithreading multiprocessing~getompthreads()
 getopt(STR,POS)~Gibt den Wert eines Parameters in der Zeichenkette STR, dessen Wert an der Stelle POS beginnt, als Zeichenkette zurück~parameter werte argumente options commandline values getopt()~getopt(
+get_utc_offset()~Gibt die Zeitzone zur�ck~get_utc_offset() time zone utc~get_utc_offset(

```

Comment:
Encoding problem. You want to update your NDB tools and also convert this file to UTF-8